includes/init_includes/init_add_crumbs.php

```diff
@@ -47,33 +45,33 @@
     "SELECT *
        FROM " . TABLE_GET_TERMS_TO_FILTER;
 $get_terms = $db->Execute($sql);
-while (!$get_terms->EOF) {
-    if (isset($_GET[$get_terms->fields['get_term_name']])) {
+foreach ($get_terms as $next_get_term) {
+    $next_get_term_name = $next_get_term['get_term_name'];
+    if (isset($_GET[$next_get_term_name])) {
         $sql =
-            "SELECT " . $get_terms->fields['get_term_name_field'] . "
-               FROM " . constant($get_terms->fields['get_term_table']) . "
-              WHERE " . $get_terms->fields['get_term_name'] . " =  " . (int)$_GET[$get_terms->fields['get_term_name']];
-        $get_term_breadcrumb = $db->Execute($sql);
-        if ($get_term_breadcrumb->RecordCount() > 0) {
-            $breadcrumb->add($get_term_breadcrumb->fields[$get_terms->fields['get_term_name_field']], zen_href_link(FILENAME_DEFAULT, $get_terms->fields['get_term_name'] . "=" . $_GET[$get_terms->fields['get_term_name']]));
+            "SELECT " . $next_get_term['get_term_name_field'] . "
+               FROM " . constant($next_get_term['get_term_table']) . "
+              WHERE " . $next_get_term_name . " = " . (int)$_GET[$next_get_term_name];
+        $get_term_breadcrumb = $db->Execute($sql, 1);
+
+        if (!$get_term_breadcrumb->EOF) {
+            $breadcrumb->add($get_term_breadcrumb->fields[$next_get_term['get_term_name_field']], zen_href_link(FILENAME_DEFAULT, $next_get_term_name . '=' . $_GET[$next_get_term_name]));
         }
     }
-    $get_terms->MoveNext();
 }
 
 /**
- * add the products model to the breadcrumb trail
+ * add the products name to the breadcrumb trail
  */
-if (isset($_GET['products_id'])) {
+if (isset($_GET['products_id'], $cPath)) {
```

Comment:
This could be a breaking-change, right?  Granted, it's probably rare that `$cPath` won't be defined.

Perhaps if the `$cPath` is missing, we should just skip the `"&cPath=" .$cPath` part of the breadcrumb URL?

(The canonical URL of a product page doesn't include a cPath, so doesn't introduce any new URLs to worry about.)

Reply:
It _should_ be set; perhaps better to leave as-is and a PHP Warning will be issued if the `$cPath` isn't set?

Reply:
Your call, @drbyte.  I've got no skin in the game.

Reply:
Let's leave it as-is. I wanna look at some related code before making a call on it. 
No need to have this hold up the other things you're doing.

Reply:
As-is in the PR or amend the PR to remove that addition?